src/CddInterface.c

```diff
@@ -38,8 +27,8 @@ extern void dd_SetLinearity(dd_MatrixPtr, char *);
 // }
 
 // The following conversion has been taken from
-// https://github.com/gap-packages/NormalizInterface/src
-// Thanks to Sebastian Gutsche
+// https://github.com/gap-packages/NormalizInterface
+// Thanks to Max Horn
 static Obj MPZ_TO_GAPOBJ( const mpz_t x)
 {
```

Comment:
If you are willing to require GAP >= 4.10, you could also replace the implementation of this function by a single line:
```C
  return MakeObjInt((const UInt *)x->_mp_d, x->_mp_size);
```

Reply:
I will add this suggestion as a comment to the source code. I may use it when Gap >= 4.10 is required. Currently, the package requires Gap >= 4.9 https://github.com/homalg-project/CddInterface/commit/7012a0192fa081aeaa385e13c95adc75a456bc8e.